apps/examples/hello/hello_main.c

```diff
@@ -56,17 +56,131 @@
 
 #include <tinyara/config.h>
 #include <stdio.h>
-
+#include <image_390x390_rgb888.h>
```

Comment:
This is the hello world example. Please do not add any other change here. If you want to add those example, please add new example.